app/services/fan_out_on_write_service.rb

```diff
@@ -63,6 +63,7 @@ def fan_out_to_public_recipients!
   def fan_out_to_public_streams!
     broadcast_to_hashtag_streams!
     broadcast_to_public_streams!
+    broadcast_to_profile_streams!
```

Comment:
Note that this is called only when the post when `broadcastable?` is true, which is true for Public (excluding unlisted/“Quiet public”) posts that are not reblogs. This is probably not what people expect.